fhir-audit/src/main/java/com/ibm/fhir/audit/logging/impl/WhcAuditCadfLogService.java

```diff
@@ -197,8 +196,10 @@ public static CadfEvent createCadfEvent(AuditLogEntry logEntry)
         Outcome cadfEventOutCome;
 
         // Cadf does't log config, so skip
-        if ((logEntry.getEventType() != AuditLogEventType.FHIR_CONFIGDATA.value()) && logEntry.getContext() != null
-                && logEntry.getContext().getAction() != null && logEntry.getContext().getApiParameters() != null) {
+        if ((logEntry.getEventType() != AuditLogEventType.FHIR_CONFIGDATA.value())
```

Comment:
String != String. Shouldn't this use equals(...)?

Reply:
Cleaned this up by addressing Lee's well-known request

Reply:
outdated now, but AuditLogEventType is an enum and enum==enum is just fine.